appcues/src/main/java/com/appcues/ui/AppcuesCustomComponentDirectory.kt

```diff
@@ -0,0 +1,18 @@
+package com.appcues.ui
+
+import com.appcues.AppcuesCustomComponentView
+import kotlin.collections.set
+
+internal object AppcuesCustomComponentDirectory {
```

Comment:
thinking out loud, probably no reason they would need to remove a custom component from the directory?